CanvasColorSpaceProposal.md

```diff
@@ -87,6 +89,21 @@ canvas.getContext('2d', { colorSpace: "p3", pixelFormat: "10-10-10-2", linearPix
 * toDataURL/toBlob convert image data to the rec-2020 color space.
 * Images with no color profile, when drawn to the canvas, are assumed to be in the sRGB color space, and are converted to rec2020 for the purpose of the draw.
 
+#### The "rec2100pq" color space
+* Support is optional.
+* User agents that select the rec2100pq color gamut in CSS media queries must support this color space.
```

Comment:
There is a Media Query called [`rec2020`](https://drafts.csswg.org/mediaqueries-4/#color-gamut) but there is no Meda Query rec2100pq or rec2100pqn.

Do you mean that only ultrawide gamut (and HDR capable) devices must support it?

For example, if a device reports true for `p3` (so it is wide gamut) and false for `rec2020` (so it is not ultrawide gamut) it should still support content delivered in rec2020 colorspace, no? I mean, that is basically what all 4k HDR TVs do now.